### PR TITLE
Dyn 4843 watch node smaller

### PR DESCRIPTION
**Pull request by jesusalvino**
### Purpose

Fixing the UI Tests based on the new WatchNode styling

### Declarations

Check these if you believe they are true

- [x] The codebase is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [x] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ] This PR modifies some build requirements and the readme is updated

### Reviewers
@QilongTang

### FYIs
@RobertGlobant20 @filipeotero

### CODE REVIEW

**Comment by jesusalvino** [comment]
imagen

File: src/DynamoCoreWpf/Views/Preview/WatchTree.xaml.cs [hunk 1]
```diff
@@ -14,27 +14,71 @@ public partial class WatchTree : UserControl
...
+                else
+                {
+                    this.Height = minHeightSize;
+                }
+            }
+            else if (e.PropertyName == "Children")
+            {
+                if (_vm.Children != null)
+                {
+                    if (!_vm.Children[0].IsCollection)
```
Comment (pinzart90) [inline src/DynamoCoreWpf/Views/Preview/WatchTree.xaml.cs:57]: Can there be a case where `_vm.Children.Count == 0` at this point ?

File: src/DynamoCoreWpf/Views/Preview/WatchTree.xaml.cs [hunk 2]
```diff
@@ -14,27 +14,71 @@ public partial class WatchTree : UserControl
...
+                    {
+                        double requiredWidth = (_vm.Children[0].NodeLabel.Length * 7.5);
```
Comment (pinzart90) [inline src/DynamoCoreWpf/Views/Preview/WatchTree.xaml.cs:59]: Why 7.5 ?
A comment at least

Comment (jesusalvino) [reply src/DynamoCoreWpf/Views/Preview/WatchTree.xaml.cs:59]: this is the width factor for each character, it has been approved / and evidenced here https://github.com/DynamoDS/Dynamo/pull/12829, this current PR is focusing in the tests.

**Review by pinzart90 — APPROVED** [review]

**Comment by jesusalvino** [comment]
> @jesusalvino What is the real change here? Is it just [64608e0](https://github.com/DynamoDS/Dynamo/commit/64608e00316ebcbb6e048412b0f252fce50193cc)? If this is the case, next time please use a clean branch

yes only the tests / for sure sorry for the confusion

**Comment by QilongTang** [comment]
@jesusalvino Please also resolve the conflict, this is also a side effect of not using clean branch

**Comment by jesusalvino** [comment]
> @jesusalvino Please also resolve the conflict, this is also a side effect of not using clean branch

Done

**Review by QilongTang — APPROVED** [review]

**Pull request merged**